ethercat_slaves :
    servo:
        vendor_id: 0x00000a88
        alias : 0
        position: 0
        product_code: 0x0a880001
        assign_activate: 0x0300 # what
        input_port: 0x6041
        output_port: 0x6040
    period_ns: 1000000  # what
    run_time: 360000  # what
    sync0_shift: 55000  # what
